sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -13,14 +13,20 @@
   `_OTELRESOURCE_` metrics.
   ([#41761](https://github.com/Azure/azure-sdk-for-net/pull/41761))
 
+* Fixed an issue where tags associated with Exceptions, specifically those logged using
+  [Activity.RecordException()](https://github.com/open-telemetry/opentelemetry-dotnet/blob/f69c025178216d1879e5884f0a74ab15ca759a23/src/OpenTelemetry.Api/Trace/ActivityExtensions.cs#L81),
+  were not being included.
+  Now, tags linked to an `ActivityEvent` of type exception are correctly exported as Custom Properties.
```

Comment:
> `ActivityEvent` of type exception

Its better to say "ActivityEvents following the otel convention for storing exception". In future Activity may support Exceptions as first class thing, and we surely don;t want any confusion with that. Make sure the wordings are accurately describing the change.